package.json

```diff
@@ -22,19 +22,19 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "react": "17.0.1",
-    "react-native": "0.64.2"
+    "react": "17.0.2",
+    "react-native": "0.66.4"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
     "babel-jest": "^26.6.3",
-    "detox": "^18.20.2",
+    "detox": "^19.4.2",
     "eslint": "7.14.0",
-    "jest": "^27.0.0",
-    "metro-react-native-babel-preset": "^0.64.0",
-    "react-test-renderer": "17.0.1"
+    "jest": "^26.6.3",
```

Comment:
What's the reason behind this? In recent Detox versions, we've generally applied a `jest≥27` constraint 🤔

Reply:
I was using the upgrade helper. I will bump up to 27 and see if I run into any issues.